lib/mayaUsd/ufe/UsdTransform3dHandler.h

```diff
@@ -44,9 +44,6 @@ class MAYAUSD_CORE_PUBLIC UsdTransform3dHandler : public Ufe::Transform3dHandler
 	// Ufe::Transform3dHandler overrides
 	Ufe::Transform3d::Ptr transform3d(const Ufe::SceneItem::Ptr& item) const override;
 
-private:
```

Comment:
Transform handler had a UsdTransform3d as a data member, which was therefore essentially a singleton, because there is only one handler for a given interface type in the UFE run-time.  This was a misguided optimization, as it meant that as soon as you tried to create more than one Transform3d interface, one per object for different USD objects (which is frequent), you would in fact get the same interface object attached to all USD objects, and you'd be scratching your head trying to figure out why the underlying object's transformation had changed without your making calls on that particular object.  Just create a transform 3d interface object per USD object, as needed.